qa/suites/rgw/verify/tasks/rgw_tempest.yaml

```diff
@@ -0,0 +1,74 @@
+tasks:
+- install:
+    flavor: notcmalloc
+- ceph:
+- tox:
+  - client.0
+- keystone:
+    client.0:
+      sha1: 12.0.0.0b2
+      force-branch: master
+      tenants:
+        - name: admin
+          description:  Admin Tenant
+      users:
+        - name: admin
+          password: ADMIN
+          project: admin
+      roles: [ name: admin, name: Member ]
+      role-mappings:
+        - name: admin
+          user: admin
+          project: admin
+      services:
+        - name: keystone
+          type: identity
+          description: Keystone Identity Service
+        - name: swift
+          type: object-store
+          description: Swift Service
+      endpoints:
+        - service: keystone
+          publicurl: http://127.0.1:$(public_port)s/v2.0
+        - service: swift
+          publicurl: http://127.0.0.1:7280/v1/KEY_$(tenant_id)s
+- rgw:
+    frontend_prefix: /swift
+    client.0:
+      valgrind: [--tool=memcheck]
+      extra_params: --rgw_keystone_url=http://localhost:35357 --rgw_keystone_admin_token=ADMIN --rgw_keystone_accepted_roles=admin,Member --rgw_swift_enforce_content_length=true --rgw_swift_account_in_url=true --rgw_keystone_implicit_tenants=true --rgw_keystone_accepted_admin_roles=admin --rgw_swift_versioning_enabled=true
+- tempest:
+    client.0:
+      force-branch: master
+      auth:
+        admin_username: admin
+        admin_project_name: admin
+        admin_password: ADMIN
+        admin_domain_name: Default
+      identity:
+        uri: http://127.0.0.1:5000/v2.0/
+        uri_v3: http://127.0.0.1:5000/v3/
+        admin_role: admin
```

Comment:
similarly to `--rgw_keystone_url` for `rgw.py`, i think the tempest task should read as much of this stuff from `ctx.keystone` as possible to avoid duplication